CHANGELOG.next.toml

```diff
@@ -339,3 +339,21 @@ Server SDKs now correctly reject operation inputs that don't set values for `req
 references = ["smithy-rs#1714", "smithy-rs#1342", "smithy-rs#1860"]
 meta = { "breaking" = true, "tada" = false, "bug" = true, "target" = "server" }
 author = "david-perez"
+
+[[aws-sdk-rust]]
+message = "Functions on `aws_smithy_http::endpoint::Endpoint` now return a `Result` instead of panicking."
+references = ["smithy-rs#1984", "smithy-rs#1496"]
+meta = { "breaking" = true, "tada" = false, "bug" = false }
```

Comment:
Does the missing `target` default to `"all"`?

Reply:
`target` only applies to `[[smithy-rs]]` since there's no server for the SDKs.

Reply:
Although, now that I'm taking a second look, this deserves a `[[smithy-rs]]` entry too since its in `aws_smithy_http`.